cef/CustomEditField/CustomEditField.rbbas

```diff
@@ -2263,7 +2272,7 @@ Implements MessageReceiver
 		    AutocompleteManual
 		    
 		    // ignore any control chars (includes Esc)
-		  elseif keyAsc <= 31 and keyAsc <> 13 then
+		  elseif keyAsc <= 31 and keyAsc <> 13  and  not (keyAsc = TAB_KEY and not KeepEntireTextIndented and not IndentVisually ) then
```

Comment:
Seems to make sense. But please keep the code tidy (remove the double blanks and the one before the closing paren).
Also, I'd prefer to rewrite the condition so that it uses fewer nots:

`elseif keyAsc <= 31 and keyAsc <> 13 and (keyAsc <> TAB_KEY or KeepEntireTextIndented or IndentVisually) then`